src/components/Select.js

```diff
@@ -177,8 +177,7 @@ class Select extends React.Component {
                     onClick={haltEvent(this._handleOptionClick.bind(null, option))}
                     style={Object.assign({},
                       styles.option,
-                      this.props.optionStyle,
-                      _isEqual(option, this.state.selected) ? styles.activeOption : null
```

Comment:
Is `_isEqual` being used anywhere else or could we remove this import?

Reply:
@TayteStokes it is used in a couple other spots on this file.